guides/customization/vnc.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using VNC with Coder
+description: Learn how to use a VNC with Coder
+---
+
+This guide will show you how to set up a Virtual Network Computing (VNC) system
+in Coder. Coder does not have a specific set of VNC providers it supports. Coder
+will render the VNC, as long as it is installed on the image used to create the
+environment.
+
+## Step 1: Create a Dockerfile image with a VNC provider installed
+
+When creating the image, be sure to set the following variables:
+
+```console
+HOME=/home/coder
+USER coder
+PORT 1234
+```
```

Comment:
I've added the above variables based on my review of Teffen's [VNC image build](https://github.com/cdr/enterprise-images/blob/ed66817dfa83e33132322c33bea653a8a5fdc057/images/vnc/Dockerfile.ubuntu). Please add context or correct if needed.

Reply:
these look good to me. I assume they are helping the VNC server know where to run